vault/resource_database_secret_backend_connection.go

```diff
@@ -1326,6 +1326,10 @@ func databaseSecretBackendConnectionRead(d *schema.ResourceData, meta interface{
 		d.Set("elasticsearch", getElasticsearchConnectionDetailsFromResponse(d, "elasticsearch.0.", resp))
 	case dbEngineSnowflake:
 		d.Set("snowflake", getSnowflakeConnectionDetailsFromResponse(d, "snowflake.0.", resp))
+	case dbEngineRedshift:
+		d.Set("redshift", getConnectionDetailsFromResponse(d, "redshift.0.", resp))
+	default:
+		return fmt.Errorf("unsupported database plugin: %s", resp.Data["plugin_name"].(string))
```

Comment:
May want to rephrase the error message a bit since we no longer rely on the `plugin_name`.  Maybe something like `no response handler for dbEngine ...` ?

Reply:
Fixed in 7bed606